utils/Classes/Panel.js

```diff
@@ -100,4 +184,15 @@ export default class Panel {
         });
         return potentialZIndex;
     }
+
+    getLowestZIndex() {
+        let lowestZIndex = this.getHighestZIndex();
```

Comment:
I would rewrite this to get rid of the call of `getHighestTindex`, to not iterate through the same map two times.

Reply:
You do not iterate twice as hetHighest and getLowest are called on different emit/action

Reply:
But in the getLowest you are calling getHighest. Where getHighest iterated over the map and than you iterate over it again later on.

Reply:
Oh yup true, the logic kinda need to go from highest to lowest. Do you have a suggestion? I could set highest to 1000 but it is ugly.
How would you return the highest z-index of the current map without iterating through it?